docs/Development/Start new OpenY project.md

```diff
@@ -3,23 +3,84 @@ Start new OpenY project
 
 Here you can find instructions how you can start project based on OpenY distribution.
 
+# New project from scratch based on OpenY
+
+In order to start new project from scratch, you can use [installation instructions](https://github.com/ymcatwincities/openy-project#installation) that will build your project and even add development environment.
+
+
+# Add OpenY to existing Drupal 8 project
+
```

Comment:
As a first step make sense to add an example of composer.json. Something like this:
```
1. Create your composer.json, see example:
{
    "name": "project_name",
    "description": "Project Description",
    "type": "project",
    "license": "GPL-2.0+",
    "minimum-stability": "dev",
    "prefer-stable": true,
    "config": {
        "preferred-install": "dist",
        "bin-dir": "bin/",
        "secure-http": false
    }
}
```